ui/src/containers/NodeList.js

```diff
@@ -101,11 +111,15 @@ class NodeList extends React.Component {
 
     const nodesSortedListWithRoles = nodesSortedList.map(node => {
       let roles = [];
-      if (node.control_plane) {
-        roles.push(intl.messages.control_plane);
-      }
-      if (node.workload_plane) {
-        roles.push(intl.messages.workload_plane);
+      if (node.bootstrap) {
+        roles.push(intl.messages.bootstrap);
+      } else {
```

Comment:
Why is this an 'else'?

Reply:
because we want to display 'bootstrap' instead of 'bootstrap/Control Plane'

Reply:
The bootstrap node is a control-plane node... and a bootstrap node at the same time. Over time, maybe bootstrap nodes can (after cluster expansion) be *not* part of the control-plane. So special-casing the bootstrap node, except for initial taints (which the UI doesn't set anyway!) seems counter-intuitive.

Reply:
ok it makes sense